src/App.jsx

```diff
@@ -0,0 +1,45 @@
+import { useEffect, useState } from "react";
+import { Route, Routes } from "react-router";
+
+import Header from "./components/Header/Header";
+import Home from "./components/Home/Home";
+import SideMenu from "./components/SideMenu/SideMenu";
+import Profile from "./components/Profile/Profile";
+
+import api from "@utilities/api";
+import PostItem from "@components/Home/components/PostFeed/PostItem/PostItem";
+import PulseLoader from "@components/Loader/PulseLoader";
+
+import "./App.css";
+
+function App() {
+  const [user, setUser] = useState(null);
+
+  useEffect(() => {
+    async function getUser() {
+      const fetch = await api.contact.get(1);
+      setUser(await fetch);
```

Comment:
What you've done here is overridden the existing `fetch` function with your own variable in this scope. It's fine, but I would avoid this in future. It looks like you're passing the fetch function into your set state function at a glance